guides/bors.md

```diff
@@ -0,0 +1,70 @@
+# Bors: our Merge Bot for PRs
+
+[Bors](https://github.com/bors-ng/bors-ng) is a tool that helps us manage pull requests (PR) merging.
+
+## Motivations
+
+One of our requirements to merge a PR is the branch has to be up-to-date with the `master` branch.<br>
+The only proper way to keep the Git history clear is to rebase the branch of the PR.
+
+It became tedious to always ask the contributors to rebase their branch when their branch was not up-to-date with `master` because:
+- It forces everyone to wait again before the merge
+- It is not always easy to successfully apply a rebase without any issue
+
```

Comment:
```suggestion
- Sometimes there are several PRs ready to be merged, which introduces complexity in coordination

```

Reply:
Do you want to remove all of this paraph by your sentence?